liblouis/compileTranslationTable.c

```diff
@@ -677,25 +606,35 @@ passFindCharacters(FileInfo *nested, widechar *instructions, int end,
 
 static void
 addForwardRuleWithSingleChar(FileInfo *nested, TranslationTableOffset *newRuleOffset,
-		TranslationTableRule *newRule, TranslationTableHeader *table) {
+		TranslationTableRule *newRule, TranslationTableHeader **table) {
```

Comment:
Why are you passing table as a pointer to a pointer? To make it clear that it can be modified?

Reply:
Because the table can be reallocated.